### PR TITLE
Vector2: Add __radd__

### PR DESCRIPTION
**Pull request by astronouth7303**
Just a simple thing to enable `<VectorLike> + <Vector>`.

### CODE REVIEW

File: ppb_vector/vector2.py [hunk 1]
```diff
@@ -213,6 +213,9 @@ def __add__(self: Vector, other: VectorLike) -> Vector:
             return NotImplemented
         return rtype(self.x + other_x, self.y + other_y)
 
+    def __radd__(self: Vector, other: VectorLike) -> Vector:
+        return self.__add__(other)
```
Comment (nbraud) [inline ppb_vector/vector2.py]:
```suggestion
        return self + other
```

**Comment by nbraud** [comment]
@astronouth7303 I implemented my requested changed.
Please take a look, and feel free to mark as approved by me (`bors r=nbraud`) if the changes are fine.

**Comment by nbraud** [comment]
PS: Please merge after #149; I'd rather deal with a merge conflict here rather than there.

**Comment by nbraud** [comment]
OK, rebased now that the conflicting PR is merged.

**Comment by nbraud** [comment]
@astronouth7303 OK, this should be good to go.
Please run `bors r=nbraud` if you are happy with it.

File: ppb_vector/vector2.py (hunk 1 above)
Comment (astronouth7303) [reply ppb_vector/vector2.py]: I'm not sure why this? Other than some sense you shouldn't call dunders directly?

**Comment by nbraud** [comment]
*pokes @astronouth7303.*

**Comment by astronouth7303** [comment]
bors r=nbraud

**Pull request merged**